dora/src/masm/x64.rs

```diff
@@ -1148,6 +1148,41 @@ impl MacroAssembler {
         }
     }
 
+    pub fn float_abs(&mut self, mode: MachineMode, dest: FReg, src: FReg) {
+        let (fst, snd) = if mode == MachineMode::Float32 {
+            (0, 0x7fffffff)
+        } else {
+            (0x7fffffff, -1)
```

Comment:
I think the numbers aren't correct yet, so I didn't quite get how they were derived in `float_neg` either.